Source/WebCore/dom/DeviceMotionEvent.idl

```diff
@@ -23,24 +23,21 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
  */
 
+// https://w3c.github.io/deviceorientation/#device-motion-event-api
+
 [
     Conditional=DEVICE_ORIENTATION,
     Exposed=Window
 ] interface DeviceMotionEvent : Event {
+    // FIXME: Add `constructor` by uncommenting below.
+    // constructor(DOMString type, optional DeviceMotionEventInit eventInitDict = {});
```

Comment:
We should just add constructor.

Reply:
@Ahmad-S792 did you have a chance to check this? Or do you need help. 
I'm asking because of the recent bug
https://bugs.webkit.org/show_bug.cgi?id=271891

Reply:
I haven't looked into it yet but I manage to fix another [SecureContext] issue in these though. So some progress but not directly on `constructor` side. :-)

Reply:
@karlcow - I just rebased it and added few more alignments but still working on how to add 'constructor'. Although it did progressed one or two WPT tests.